ironfist/src/cpp/init.cpp

```diff
@@ -10,6 +10,7 @@
 #include "resource/resources.h"
 #include "sound/sound.h"
 #include "town/town.h"
+#include "registry_prefs.h"
```

Comment:
Nitpick: Please alphabetize the includes in accordance with the style guide ( http://projectironfist.pbworks.com/w/page/109715005/Coding%20Guidelines )